### PR TITLE
Fixed time parsing of harvest start_date

### PR DESCRIPTION
**Pull request by spiewok**
🔑 Made it case-insensitive.

### CODE REVIEW

File: src/test/java/ch/aaap/harvestclient/core/util/FormatUtilTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import java.time.temporal.ChronoField;
+import java.time.temporal.TemporalAccessor;
+import org.junit.jupiter.api.Test;
+import retrofit2.http.FormUrlEncoded;
+
+class FormatUtilTest {
+
+
+  @Test
+  public void should_parse_lowercase_correctly() {
```
Comment (mnembrini) [inline src/test/java/ch/aaap/harvestclient/core/util/FormatUtilTest.java:16]: use camelCase for test method name

File: src/test/java/ch/aaap/harvestclient/core/util/FormatUtilTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  public void should_parse_without_correctly() {
+
+    String timeString = "12:01";
+    DateTimeFormatter formatter = FormatUtil.getDateTimeFormatter(false);
+    TemporalAccessor time = LocalTime.parse(timeString, formatter);
+
+    assertEquals(12, time.get(ChronoField.HOUR_OF_DAY));
+  }
+
+}
```
Comment (mnembrini) [inline src/test/java/ch/aaap/harvestclient/core/util/FormatUtilTest.java:45]: missing newline

File: README.md [hunk 3]
```diff
@@ -74,6 +116,14 @@ compile 'ch.aaap:harvest-client:$version'
     ```   
 * Run ```gradle build``` in the root directory of the repository
 
+### Creating a test Harvest account
+go through creating a new account.
+My Profile -> link under work email to your Harvest ID settings -> Developers -> create new personal Token (name does not matter)
```
Comment (simone-montemezzani-3AP) [inline README.md:121]: link should work, right? https://id.getharvest.com/developers

Comment (mnembrini) [reply README.md:121]: yes that's the current link

**Comment by mnembrini** [comment]
Current status: CI tests are not passing because I need to setup new test accounts.
Also CI release process needs to be updated to release to github registry.
Haven't had time to work on that yet

**Comment by mnembrini** [comment]
This PR is released as snaphot here https://github.com/3AP-AG/harvest-client/packages/261886?version=1.1.2.001
with version 1.1.2.001

**Pull request merged**